.pre-commit-config.yaml

```diff
@@ -37,6 +37,9 @@ repos:
     hooks:
     - id: black
       language_version: python3
+    - id: black-jupyter
+      language_version: python3
```

Comment:
`black-jupyter` does everything `black` does, but also checks notebooks - you probably just need that one

Reply:
While you're at it @cphyc, you can remove the `language_version` line. It became useless in the latest release.